File: LoisirsVF/lib/model/Loisir.dart
class Loisir{
  String id_loisir,nom,description,latitude,longitude,type,capacite;

  Loisir({
    this.id_loisir,
    this.nom,
    this.description,
    this.latitude,
    this.longitude,
    this.type,
    this.capacite
  });
  factory Loisir.fromJson(Map<String, dynamic> json) {
    return Loisir(
        id_loisir: json['id_loisir'] as String,
        nom: json['nom'] as String,
        description: json['description'] as String,
        latitude: json['latitude'] as String,
        longitude: json['longitude'] as String,
        type: json['type'] as String,
        capacite: json['capacite'] as String
    );
  }
}

File: loisir/app/Models/Loisir.php
<?php

/**
 * Created by Reliese Model.
 */

namespace App\Models;

use Illuminate\Database\Eloquent\Collection;
use Illuminate\Database\Eloquent\Model;

/**
 * Class Loisir
 * 
 * @property int $id_loisir
 * @property string $nom
 * @property string $description
 * @property string $latitude
 * @property string $longitude
 * @property string $type
 * @property int $capacite
 * 
 * @property Collection|Photo[] $photos
 *
 * @package App\Models
 */
class Loisir extends Model
{
	protected $table = 'loisir';
	protected $primaryKey = 'id_loisir';
	public $timestamps = false;

	protected $casts = [
		'capacite' => 'int'
	];

	protected $fillable = [
		'nom',
		'description',
		'latitude',
		'longitude',
		'type',
		'capacite'
	];

	public function photos()
	{
		return $this->hasMany(Photo::class, 'id_loisir');
	}
}
